test/global_test_setup.ts

```diff
@@ -0,0 +1,11 @@
+global.console = {
```

Comment:
deeply minor nit; mind if we kebab-case this instead of snake-case?

Reply:
oh yeah np, hangover from most recent python project. changed and merging